scripts/generate-CHANGES.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2009-2016 The Mumble Developers. All rights reserved.
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file at the root of the
+# Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+# Convert the Git log of the repository into a CHANGES file.
+#
+# This script uses the same formatting that
+# 'git2cl.pl' has historically used.
+
+from __future__ import (unicode_literals, print_function, division)
+
+import subprocess
+import sys
+import re
+
+def textwidth(text):
+	'''
+		Calculate the with of 'text' in number
+		of characters.
+
+		This function counts tabs as 8 characters.
+	'''
+	count = 0
+	for char in text:
+		if char == '\t':
+			count += 8
+		else:
+			count += 1
+	return count
+
+
+def perlCompatibleTextWrap(text, width, initial_indent, subsequent_indent):
```

Comment:
I don't think so.  The code becomes harder to read (to me, at least -- I tried).

Comments talk about the width, but the variable is named "columns".
The `textwidth` function has width in the name. Should it be called `countTextColumns`?
How to you discuss about column count in the code? "Width"?

I think it's more readable to just use width. :-)